src/web/mjs/connectors/Mangaxin.mjs

```diff
@@ -0,0 +1,22 @@
+import WordPressEManga from './templates/WordPressEManga.mjs';
+
+/**
+ *
+ */
+export default class MangaXin extends WordPressEManga {
+
+    /**
+     *
+     */
+    constructor() {
+        super();
+        super.id = 'mangaxin';
```

Comment:
The id changed, so users will loose all their bookmarks they made for `mangaceng` is this intended?

Reply:
Yeah... The website itself changed everything (Including their logo).. So, it's as if the MangaCeng was shut down and then, a new website called MangaXin was started. I dealt it as if it was a new site.

Reply:
Bookmarks are probably still working and should be kept, therefore the ID should be `mangaceng`.

The ID is just a unique identifier with no relation to the website name, in fact it could be a random string as well. So there is no harm when it differs from the class or view name.